public/components/ContentPanel/ContentPanel.tsx

```diff
@@ -52,7 +52,7 @@ const ContentPanel = (props: ContentPanelProps) => (
       <EuiFlexItem>
         {typeof props.title === 'string' ? (
           <EuiTitle
-            size={props.titleSize || 'l'}
+            size={props.titleSize || 's'}
```

Comment:
why change the default value here, instead of passing `s` to `size`?

Reply:
This is from ux designer's comments